nimble/host/audio/include/audio/ble_audio.h

```diff
@@ -562,6 +583,66 @@ struct ble_audio_event_codec_unregistered {
     const struct ble_audio_codec_record *record;
 };
 
+/** @brief BASS Source Removed */
+struct ble_audio_event_bass_remote_scan {
+    /** Connection Handle of Broadcast Assistant that is performing Scan procedure for us */
+    uint16_t conn_handle;
+};
+
+/** @brief BASS Source Added */
+struct ble_audio_event_bass_source_added {
+    /** Source ID */
+    uint8_t source_id;
+
+    /** Event status. 0 on success, BLE_HS Error code otherwise */
+    uint8_t status;
+};
+
+/** @brief BASS Source Modify */
+struct ble_audio_event_bass_source_modify {
```

Comment:
```suggestion
struct ble_audio_event_bass_source_modified {
```

to follow the naming:
`ble_audio_event_bass_source_added`
`ble_audio_event_bass_source_removed`